job-dsls/jobs/kie/kogito/kogito_tooling_prerelease_branch_UMB_trigger.groovy

```diff
@@ -0,0 +1,98 @@
+/**
+ * Creates job that triggers when a new `<MAJOR>-<MINOR>-<PATCH>-prerelease` branch is
+   pushed to kogito-tooling repository and notifies QE of this by sending and UMB message.
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+def repo = "kogito-tooling"
+def repoBranch = ''
+def ghOrgUnit = "kiegroup"
+def ghAuthTokenId = "kie-ci"
+def labelName = "kie-rhel7"
+def regexpFilterRegexValue = '^([0-9]+)\\.([0-9]+)\\.([0-9]+)-prerelease$'
```

Comment:
Just wondering if the `$` needs to be escaped as well, can you please check?

Reply:
I removed them from the regex for now, but tests did not fail on this